src/main/java/com/Configuration/AppAuthenticationSuccessHandler.java

```diff
@@ -0,0 +1,17 @@
+package com.Configuration;
+
+import org.springframework.security.core.Authentication;
+import org.springframework.security.web.authentication.SimpleUrlAuthenticationSuccessHandler;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class AppAuthenticationSuccessHandler extends SimpleUrlAuthenticationSuccessHandler {
```

Comment:
is it normal that there is no implementation in this method?

Reply:
why not just instantiate and return a SimpleUrl...() in the @Bean declaration?

Reply:
I will change to what you suggest here.